LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWLaunchDelayedMission.uc

```diff
@@ -47,6 +47,74 @@ simulated function InitScreen(XComPlayerController InitController, UIMovie InitM
 	BuildScreen();
 }
 
+// Part of the fix for issue #961. This function makes sure that the "BOOST INFILTRATION", "VIEW SQUAD", "ABORT", "LAUNCH MISSION", and "WAIT"
+// buttons are set up properly for controller and keyboard support. The most important thing it does is make sure the buttons appear as normal
+// buttons rather than unselectable hotlinks when the controller is active. This way we can have a single column of buttons which can be selected
+// via the D-Pad or arrow keys and "clicked" with the A button or Enter/Spacebar keys.
+//
+// KDM : UIMission --> BindLibraryItem() code has been copied and the following modifications have been made :
+// 1.] All UIButtons have been given the style eUIButtonStyle_NONE so they appear as normal buttons when using a controller.
+// 2.] The delegate, OnSizeRealized, is no longer set for any of the UIButtons since it pointed to the empty function OnButtonSizeRealized().
+// 3.] Buttons are setup identically whether using a controller or mouse & keyboard; more specifically, positioning, width, and gamepad icons
+//		are no longer set when using a controller.
+// 4.] Any navigation setup has been removed since it should all be dealt with in RefreshNavigation().
+// 5.] The Ignore button is now created in this function along with all the other UIButtons. For more information please read the comments in BuildOptionsPanel(). 
+simulated function BindLibraryItem()
+{
+	local Name AlertLibID;
+	
+	AlertLibID = GetLibraryID();
+	
+	if (AlertLibID != '')
+	{
+		LibraryPanel = Spawn(class'UIPanel', self);
+		LibraryPanel.bAnimateOnInit = false;
+		LibraryPanel.InitPanel('LibraryPanel', AlertLibID);
+
+		ButtonGroup = Spawn(class'UIPanel', LibraryPanel);
+		ButtonGroup.InitPanel('ButtonGroup', '');
+
+		// KDM : Boost infiltration button
+		Button1 = Spawn(class'UIButton', ButtonGroup);
+		Button1.ResizeToText = false;
+		Button1.InitButton('Button0', "", , eUIButtonStyle_NONE);
+		
+		// KDM : View squad button
+		Button2 = Spawn(class'UIButton', ButtonGroup);
+		Button2.ResizeToText = false;
+		Button2.InitButton('Button1', "", , eUIButtonStyle_NONE);
+
+		// KDM : Abort button
+		Button3 = Spawn(class'UIButton', ButtonGroup);
+		Button3.ResizeToText = false;
+		Button3.InitButton('Button2', "", , eUIButtonStyle_NONE);
+		
+		// KDM : The Confirm button is now given the style eUIButtonStyle_NONE so that, when using a controller, it doesn't appear as a hotlink.
```

Comment:
Wouldn't this have been a problem in WOTC too? Or does the LWOTC launch screen cause specific issues?

Reply:
Haven't played base XCom 2 in quite a while; however, based on recollection and looking at the code :

For something like a Guerrilla Ops, UIMission_GOps, the only numbered button to be used was Button1;  Button2 and Button3 are hidden in UpdateButtons(). Button 1 is a normal button which just displays the region or Guerrilla Ops title. Meanwhile, the confirm and ignore buttons are hotlinks which link to, something like, "Go to mission" and "Cancel mission".